source/playbooks/playbook-permissions.rst

```diff
@@ -0,0 +1,74 @@
+
+* **System Scheme:** Applies permissions universally across all teams and channels.
+* **Team Override Schemes:** Allow admins to customize permissions for each team (available in Mattermost Enterprise and Professional).
+
+You can set the default permissions granted to System Admins, Team Admins, Channel Admins, Playbook Admins, Guests (if enabled), and All Members. The permissions granted in the System Scheme apply system-wide, meaning:
+
+* **Guests:** If Guest Accounts are enabled, permissions apply to guest users in all channels, in all teams.
+* **All Members:** Permissions apply to all members, including Admins, in all channels, in all teams.
+* **Channel Administrators:** Permissions apply to all Channel Admins in all channels, in all teams.
+* **Playbook Admins:** Permissions apply to all Playbook Admins in all channels, in all teams.
+* **Team Administrators:** Permissions apply to all Team Admins, in all teams.
+
+For more information about System and Team Override Schemes, refer to the `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation.
+
+Roles
+-----
+
+**Member**
+
+In the context of Playbooks, members are users of Mattermost who belong to the team within which the run is active and who may have a vested interest in the run. Depending on the permissions granted, members can have the same privileges as Playbook Admins or restricted privileges. Members are also referred to as participants in a run.
+
+**Playbook Admin**
+
+In the context of Playbooks, Playbook Admins are also members, but have elevated permissions to change playbook and run visibility and functional settings. The Playbook Admin role is a default role, and members need to be promoted to the role from within Playbooks. The Playbook Admin role then applies across the Playbooks platform. They do not have access to the System Console and their privileges are managed by System Admin.
+
+Manage playbook visibility
+--------------------------
+
+The default Playbooks settings are completely open. This means that members can add and remove other members, edit playbooks, convert Public playbooks to Private playbooks and so on. The default settings allow all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management.
+
+Create read-only playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example, set permissions to only allow Playbook Admins to edit playbooks and start runs. Team members can view Public and Private playbooks and runs but can't edit playbooks, check off tasks, or remove members from runs.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
```

Comment:
This would also restrict members from creating public/private playbooks. Not sure that's the intent here. 
To avoid this you would have to unselect the individual permissions except the creation ones.